app.server/config/config.ts

```diff
@@ -1,3 +1,27 @@
+import * as dotenv from "dotenv";
+
+function getIntFromEnvVariable(env: string | undefined, defaultValue: number) : number {
+  try {
+    if (env !== undefined) {
+      return Number.parseInt(env);
+    }
+  } finally {
+    return defaultValue;
+  }
+}
+
+dotenv.config();
+
+export const POSTGRES_HOST = process.env.POSTGRES_HOST;
+export const POSTGRES_PORT = getIntFromEnvVariable(process.env.POSTGRES_PORT, 5432);
+export const POSTGRES_USER = process.env.POSTGRES_USER;
+export const POSTGRES_PASSWORD = process.env.POSTGRES_PASSWORD;
+export const POSTGRES_DATABASE = process.env.POSTGRES_DATABASE;
+export const POSTGRES_SSL = process.env.POSTGRES_SSL === 'true' ? true : false;
+
+export const CATALOG_API_URL = `${process.env.CATALOG_API_PROTOCOL}://${process.env.CATALOG_API}`;
+
+export const GEOSERVER_URL = `${process.env.GEOSERVER_PROTOCOL}://${process.env.GEOSERVER_HOST}`
 
 /** This file gets overwritten in a production build with one of the
```

Comment:
It doesn't anymore, I commented out the code on deploy as its honestly horrendously confusing to work with at dev time, should probably strip that comment though

Reply:
Not sure what's *that* confusing...  but we might need to look at this one over a cup of tea. ☕️  😄 

Having multiple "tenants" in any app make is going to make it much more complicated, like internationalisation or something - it's a whole extra layer of indirection. 

But this change isn't a finished solution to that. The multi-tenancy would either need to be 

1. all removed so that the Deli becomes singularly only for the Lidar, or
2. left as it is so the code is consistent, or
3. solved with a new solution to the problem.

We haven't decided that we definitely will never make another site on the Deli so it would be wrong to do (1). It would also be time-consuming to do either (1) or (3). 

Either way, currently the changeset code is incorrect - the comment `/** This file gets overwritten in a production build...` isn't true. Better solutions welcome, but need to be consistent / correct.

Reply:
The issue here is that config logic is required in all environments now, as well as at dev time, it feels like a bad idea to have to essentially copy paste that logic into three separate places

Reply:
I have pushed the logic back into the tennants, not hugely happy with it but compile should work as before now

Reply:
This should hopefully solved in #52